language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/backend/java/inner/InterfaceClass.scala

```diff
@@ -38,7 +38,6 @@ object InterfaceClass extends StrictLogging {
               interfaceName,
               packagePrefixes,
               interface.choices,
-              withPrefixes = false, // TODO: remove in #15154
```

Comment:
Does this diff incorporate part of #15215?

Reply:
yes it is. it will be gone when i merge the main into this PR